### ISSUE TITLE
Python Error when running make

### ISSUE THREAD

**Issue by pxanthopoulos** [unikraft/app-duktape#5]
When running `make`, the following error appears:
``` 
File "path/to/workdir/apps/app-duktape/build/duktape/origin/duktape-2.4.0/tools/configure.py", line 139, in get_duk_version
    m = r.match(line)
TypeError: cannot use a string pattern on a bytes-like object
```
I'm using the Makefiles from the following pull request:
https://github.com/unikraft/app-duktape/pull/4/files

(hackathens23 team name: SheepDog)

**Comment by razvand** [comment]
Hackathon points: 5

**Comment by JoachimBose** [comment]
So the problem is that the python scripts ran by make are made for 2.7 and not python 3, we updated the readme with instructions in this pr: #6

**Comment by razvand** [comment]
Solved by #6 and #9
